core/block/editor/spaceview.go

```diff
@@ -163,6 +163,18 @@ func (s *SpaceView) SetSpaceData(details *types.Struct) error {
 	}
 
 	if changed {
+		if st.ParentState().ParentState() == nil {
```

Comment:
maybe add check that the first ParentState is not nil?

Reply:
it's not possible because we do st := s.NewState() 10 lines above :)